### PR TITLE
Update the content of Package.md and README.md file & Replace icon with high resolution image

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
$subject related to https://github.com/ballerina-platform/ballerina-standard-library/issues/2657

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Package.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+Represents the client OAuth2 provider, which is used to generate OAuth2 access tokens using the configured OAuth2 token endpoint configurations. This supports the client credentials grant type, password grant type, and refresh token grant type.
 
-### Report Issues
+## Report Issues
 
-To report bugs, request new features, start new discussions, view project boards, etc., go to the <a target="_blank" href="https://github.com/ballerina-platform/ballerina-standard-library">Ballerina standard library parent repository</a>.
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina standard library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
 
-### Useful Links
+## Useful Links
```
Comment (shafreenAnfar) [inline ballerina/Package.md:21]: Don't we need the Docs and BBE links?

**Review by shafreenAnfar — APPROVED** [review]

File: ballerina/Package.md (hunk 1 above)
Comment (ldclakmal) [reply ballerina/Package.md:21]: We only have these in the `README.md`, but not in the `Package.md`. Right? If it is needed, I will add those to `Package.md` as well.

Comment (shafreenAnfar) [reply ballerina/Package.md:21]: Yeah, we don't need it. We can add those later as central evolves.

Comment (ldclakmal) [reply ballerina/Package.md:21]: Ack.

**Pull request merged**